moveit_simple/src/moveit_simple.cpp

```diff
@@ -88,7 +88,7 @@ bool Robot::addTrajPoint(const std::string & traj_name, const Eigen::Affine3d po
     Eigen::Affine3d pose_rel_robot = transformToBase(pose, frame);
     std::unique_ptr<TrajectoryPoint> point =
       std::unique_ptr<TrajectoryPoint>(new CartTrajectoryPoint(pose_rel_robot, time, point_name));
-    success = addTrajPoint(traj_name, point);
+//    success = addTrajPoint(traj_name, point);
```

Comment:
I haven't yet changed this part of the code. It was done to see if other calls work.